src/modules/tests/test-data/cat-adi-part2/manoeuvres/__tests__/manoeuvre.reducer.spec.ts

```diff
@@ -0,0 +1,3 @@
+describe('Manoeuvres Cat ADI2 Reducer', () => {
+
+});
```

Comment:
Please add tests - or . a TODO to indicate that tests will be added.

Reply:
but I'd prefer actual tests as there is always the danger that tests don't get done

Reply:
The reducer only has a default return state currently, so this will need to be addressed when the manoeuvres modal is implemented. In Cat ADI2 it's a big shift from how it has currently been done.